cms/djangoapps/contentstore/tests/test_permissions.py

```diff
@@ -2,7 +2,6 @@
 Test CRUD for authorization.
 """
 import copy
-import mock
```

Comment:
Odd, not sure where that came from.. could have sworn I hit an "unused import" violation.. Will add back.